.CI/create_feedstocks.py

```diff
@@ -254,10 +252,24 @@ def write_token(name, token):
                 team.url + "/memberships/" + new_member
             )
 
+    # Update status based on remote
+    subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
+    subprocess.check_call(['git', 'pull', '--rebase', '--autostash'])
```

Comment:
However there were cases where that would fail due to a merge conflict. Followed it up with PR ( https://github.com/conda-forge/staged-recipes/pull/1473 ), which stashes everything not in the index. Then allows rebase to handle the merge and remaining files from the index. That actually seems to be the most robust iteration yet. Hoping that is the last one though.